### ISSUE TITLE
Run parameter sweep

### ISSUE THREAD

**Issue by Brinoleum** [Brinoleum/ABCD-caduceus#1]
Current parameters are based on the research paper the model is based on, calling for learning rate = 8e-3 and cosine decay. Training run logs on W&B show this doesn't converge very quickly, if at all. We need to sweep over various hyperparameters to see which configuration gets the best results on the model.

**Comment by Brinoleum** [comment]
depends on parallelization because I also want to experiment with the batch size, among other configs, which necessitates sharding since the A40 can only fit so much at a time

**Comment by Brinoleum** [comment]
hyperparam sweep is inconclusive, seems like there's a more fundamental error with the model (either the pre-training or the classification head), need to look more into that before deciding which params are best
